### PR TITLE
11 add django admin to manage api keys

### PR DESCRIPTION
**Pull request by koladev32**
Related to issue #11

### CODE REVIEW

File: rest_framework_simple_api_key/admin.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+    list_filter = ("entity", "revoked", "expires_at")
+
+    def get_readonly_fields(
+        self, request: HttpRequest, obj: APIKey = None
+    ) -> typing.Tuple[str, ...]:
+        fields = ()
+
+        if obj and obj.revoked:
+            fields += ("name", "revoked", "expires_at")
```
Comment (Ederene20) [inline rest_framework_simple_api_key/admin.py:20]: Shouldn't we also include the entity field?

File: rest_framework_simple_api_key/admin.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    def save_model(
+        self,
+        request: HttpRequest,
+        obj: APIKey,
+        form: typing.Any = None,
+        change: bool = False,
+    ) -> None:
+
+        if not obj.pk:
```
Comment (Ederene20) [inline rest_framework_simple_api_key/admin.py:32]: Why `if not obj.pk`? What are you trying to do? Explain, please.

File: rest_framework_simple_api_key/admin.py (hunk 1 above)
Comment (koladev32) [reply rest_framework_simple_api_key/admin.py:20]: Yes

File: rest_framework_simple_api_key/admin.py (hunk 2 above)
Comment (koladev32) [reply rest_framework_simple_api_key/admin.py:32]: I'll add better comments then. The goal is: if the object is been created, then we need to display the `api_key` value.

**Pull request merged**